plugins/catalog-backend-module-ldap/src/ldap/client.ts

```diff
@@ -40,8 +40,12 @@ export class LdapClient {
     logger: Logger,
     target: string,
     bind?: BindConfig,
+    tls?: TLSConfig,
   ): Promise<LdapClient> {
-    const client = ldap.createClient({ url: target });
+    const client = ldap.createClient({
```

Comment:
Does setting `NODE_TLS_REJECT_UNAUTHORIZED` achieve the same thing? Thinking this might not be the level we want to configure this at.

Reply:
Yes, but you disable TLS certificate verification for all connections, I don't want this in a production environment.